packages/projection-typeorm/src/operators/storeStakePoolMetadataJob.ts

```diff
@@ -20,7 +20,13 @@ export const storeStakePoolMetadataJob = typeormOperator<Mappers.WithStakePools
         })
       );
     for (const task of tasks) {
-      await pgBoss.send(STAKE_POOL_METADATA_QUEUE, task, { slot: header.slot });
+      await pgBoss.send(STAKE_POOL_METADATA_QUEUE, task, {
+        expireInHours: 48,
+        retryBackoff: true,
+        retryDelay: 120,
+        retryLimit: 10,
```

Comment:
What happens when the limit is reached? If this is a terminal state, we either need a way to trigger retries or just keep trying with significant delay